### PR TITLE
Add example snippet for makeOffer marketplace direct

### PR DESCRIPTION
**Pull request by jarrodwatts**

### CODE REVIEW

**Review by joaquim-verges — APPROVED** [review]

File: src/core/classes/marketplace-direct.ts [hunk 1]
```diff
@@ -274,6 +294,21 @@ export class MarketplaceDirect {
...
+   * Accept an offer on a direct listing
+   *
+   * @remarks Accept an offer on a direct listing
+   *
+   * @example
+   * ```javascript
+   * // The listing ID of the asset you want to bid on
+   * const listingId = 0;
+   * // The price you are willing to bid for a single token of the listing
+   * const offeror = "0x...";
```
Comment (jnsdls) [inline src/core/classes/marketplace-direct.ts:307]: the description and the variable don't line up here

File: src/core/classes/marketplace-direct.ts [hunk 2]
```diff
@@ -215,6 +215,26 @@ export class MarketplaceDirect {
...
+   * const quantity = 1;
+   * // The address of the currency you are making the offer in (must be ERC-20)
+   * const currencyContractAddress = NATIVE_TOKENS[ChainId.Rinkeby].wrapped.address
+   *
+   * await contract.direct.makeOffer(
+   *   listingId,
+   *   quantity,
+   *   currencyContractAddress,
+   *   bidAmount
+   * );
```
Comment (jnsdls) [inline src/core/classes/marketplace-direct.ts:236]: we define `pricePerToken` but then pass `bidAmount` here?

File: src/contracts/marketplace.ts [hunk 3]
```diff
@@ -357,7 +364,7 @@ export class Marketplace implements UpdateableNetwork {
    * @example
    * ```javascript
    * // the bid buffer in basis points
-   * const bufferBps = 500;
+   * const bufferBps = 500; // 5%
```
Comment (jnsdls) [inline src/contracts/marketplace.ts:367]:
```suggestion
   * const bufferBps = 5_00; // 5%
```

File: src/contracts/marketplace.ts [hunk 4]
```diff
@@ -196,6 +196,13 @@ export class Marketplace implements UpdateableNetwork {
    *
    * @param listingId - the listing id
    * @returns either a direct or auction listing
+   *
+   * @remarks Create and manage auctions in your marketplace.
```
Comment (jnsdls) [inline src/contracts/marketplace.ts:200]: this remark doesn't make sense, this is the remark on the `getListing()` function?

File: src/contracts/marketplace.ts [hunk 5]
```diff
@@ -196,6 +196,13 @@ export class Marketplace implements UpdateableNetwork {
...
+   * @example
+   * ```javascript
+   * const listingId = "1";
+   * const listing = await contract.getListing(listingId);
```
Comment (jnsdls) [inline src/contracts/marketplace.ts:204]:
```suggestion
   * const listing = await contract.getListing(1);
```

File: src/core/classes/marketplace-direct.ts [hunk 6]
```diff
@@ -215,6 +215,26 @@ export class MarketplaceDirect {
...
+   * @example
+   * ```javascript
+   * // The listing ID of the asset you want to offer on
+   * const listingId = 0;
+   * // The price you are willing to offer per token
+   * const pricePerToken = 1;
+   * // The quantity of tokens you want to receive for this offer
+   * const quantity = 1;
+   * // The address of the currency you are making the offer in (must be ERC-20)
+   * const currencyContractAddress = NATIVE_TOKENS[ChainId.Rinkeby].wrapped.address
```
Comment (jnsdls) [inline src/core/classes/marketplace-direct.ts:229]: where is `NATIVE_TOKENS` coming from here?

**Review by jnsdls — CHANGES_REQUESTED** [review]
@jarrodwatts couple of things to fix

File: src/core/classes/marketplace-direct.ts (hunk 1 above)
Comment (jarrodwatts) [reply src/core/classes/marketplace-direct.ts:307]: wdym? they are both 1 space from the *

Comment (jnsdls) [reply src/core/classes/marketplace-direct.ts:307]: @jarrodwatts no like... read the description, and then look at what the variable that you're declaring is

File: src/contracts/marketplace.ts (hunk 5 above)
Comment (jarrodwatts) [reply src/contracts/marketplace.ts:204]: the way i did it is consistent with other example snippets we have and IMO more clear what is happening

**Pull request merged**